Lib/asyncio/mixins.py

```diff
@@ -0,0 +1,23 @@
+"""Event loop mixins."""
+
+__all__ = ('LoopBoundedMixin',)
```

Comment:
Make sense, I just thought that maybe it would useful for a third-patry code to have the ability to create classes that can be automatically bound to `EventLoop` instance.